### PR TITLE
feat(EmergencyMeetingCommand): add emergency meeting command

### PR DESCRIPTION
**Pull request by Beatzoid**
A command to generate an Emergency Meeting meme. Example: https://i.imgur.com/PLIAcmr.png

### CODE REVIEW

**Review by Milo123459 — APPROVED** [review]

**Comment by Milo123459** [comment]
I'll look more into vacefron in the morning. Looks good but that's me on my phone

**Comment by Beatzoid** [comment]
Thanks! <3

File: src/client/Client.ts [hunk 1]
```diff
@@ -31,6 +32,7 @@ class Spencer extends Client {
 	public prefix: string;
 	public owners: Array<string>;
 	public config: Config;
+	public vacefron: VACEFronJS;
```
Comment (Milo123459) [inline src/client/Client.ts:35]: initiate the class here

Comment (Beatzoid) [reply src/client/Client.ts:35]: Do you mean like `public vacefron: VACEFronJS = new VACEFronJS()`?

Comment (Milo123459) [reply src/client/Client.ts:35]: Yes

File: src/commands/Fun Commands/EmergencyMeetingCommand.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { MessageAttachment } from 'discord.js';
+import { RunFunction } from '../../interfaces/Command';
+
+export const run: RunFunction = async (client, message, args) => {
+	if (!args.length)
+		return message.channel.send(
+			'I need some text to generate this meme! `emergencymeeting <text>`'
+		);
+	const buffer = await client.vacefron.emergencyMeeting(args.join(' '));
+	const attachment = new MessageAttachment(buffer, 'emergencymeeting.png');
```
Comment (Milo123459) [inline src/commands/Fun Commands/EmergencyMeetingCommand.ts:10]: Is this going to write to a file?

Comment (Beatzoid) [reply src/commands/Fun Commands/EmergencyMeetingCommand.ts:10]: It didn't on my local machine

File: src/client/Client.ts (hunk 1 above)
Comment (Beatzoid) [reply src/client/Client.ts:35]: Alright

File: src/commands/Fun Commands/EmergencyMeetingCommand.ts (hunk 2 above)
Comment (Milo123459) [reply src/commands/Fun Commands/EmergencyMeetingCommand.ts:10]: Let me look into MessageAttachment.

Comment (Milo123459) [reply src/commands/Fun Commands/EmergencyMeetingCommand.ts:10]: Upon looking into it, the second argument can be undefined / null, can you try without the second param and see if it still functions?

Comment (Beatzoid) [reply src/commands/Fun Commands/EmergencyMeetingCommand.ts:10]: Yes I will try that

Comment (Milo123459) [reply src/commands/Fun Commands/EmergencyMeetingCommand.ts:10]: Let me know of the results.

Comment (Beatzoid) [reply src/commands/Fun Commands/EmergencyMeetingCommand.ts:10]: It does work without the second arg, do you want me to commit the changes?

Comment (Milo123459) [reply src/commands/Fun Commands/EmergencyMeetingCommand.ts:10]: Yes please.

**Comment by Milo123459** [comment]
LGTM.

**Pull request merged**